docs/tutorials/ofo.md

```diff
@@ -0,0 +1,113 @@
+---
+sidebar_position: 2
+id: ofo
+title: Cloud native flags with the OpenFeature Operator
+---
+
+# Cloud native feature-flagging with the OpenFeature Operator
+
+In the following tutorial, we'll see how to leverage _flagd_ and the OpenFeature Operator to enable cloud-native, self-hosted feature flags in your Kubernetes cluster. [flagd](https://github.com/open-feature/flagd) is a "feature flag daemon with a Unix philosophy". Put another way, it's a small, self-contained binary that evaluates feature flags, uses standard interfaces, and runs just about anywhere. It can be deployed in a central location serving multiple clients or embedded into a unit of deployment (such as a pod in Kubernetes). The [OpenFeature Operator](https://github.com/open-feature/open-feature-operator) is a K8s-flavored solution for easily adding flagd to any relevant workloads. It parses Kubernetes spec files and adds flagd and associated objects to the workloads based on annotations and custom resource definitions it understands.
+
+## Let's do it!
+
+### Prerequisites
+
+- If you don't have access to an existing K8s cluster, we recommend you install [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) for this tutorial. _kind_ is is similar to minikube (another solution for running a cluster locally you may be familiar with) but supports more than one node, so it makes for a slightly more realistic experience.
```

Comment:
Can we have the language more like:

"MicroK8s and K3s are easily installable Kubernetes clusters you can use locally, we recommend using one of these. The benefit of these is that they are the same as you would use in a production environment. If you would prefer something simpler _kind_ is a local docker-based Kubernetes option."

Reply:
Hmm... Personally, I would recommend `kind` for something like this.

I have a cluster running on my home server which I installed with `kubeadm` but I don't use it for testing, generally. For testing and demos, I always use local ephemeral clusters with `kind`, and then blow them away when my testing is done.

I'll add some language that doesn't necessarily recommend either, but presents both options neutrally.